src/parse/ast/mod.rs

```diff
@@ -69,6 +69,8 @@ pub enum Node {
     EqOp,
     LeOp,
     GeOp,
+    BLShiftOp,
```

Comment:
Do we actually need all these `Op` `Node`s?
Can't this be a separate enum? (like `NodeOp`?)
Pretty sure these can't be top-level constructs anwyay in the generation stage, but need to check this.